### PR TITLE
improve description

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/133

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOSiteRename.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ ms.reviewer:
 > [!NOTE]
 > This Feature is part of the Admin Center Preview. If your tenant is not part of the Admin Center Preview, you will get an error when trying to run this cmdlet.
 
-Starts a job to rename a site. You can change the title, the URL or both of a particular site on a SharePoint Online collection
+Starts a job to rename a site. You can change the URL, and optionally the site title along with changing the URL, of a particular site on a SharePoint Online collection
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Start-SPOSiteRename.md:20]:
```suggestion
Starts a job to rename a site. You can change the URL, and optionally the site title along with changing the URL, of a particular site on a SharePoint Online collection.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by dariomws — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOSiteRename.md (hunk 1 above)
Comment (tseward) [reply sharepoint/sharepoint-ps/sharepoint-online/Start-SPOSiteRename.md:20]: I would drop the word 'particular'.

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**